docs/functional-areas/lighthouse/system_overview.md

```diff
@@ -29,7 +29,7 @@ The protocol for the lighthouse V1 is composed of frames starting with sync puls
 
 The protocol in lighthouse 2 does not use the same frame concept and there is no need for frame sync.
 
-The lighthouse V2 protocol supports more than 2 base stations. There's currently experimental support implemented for 2+ base stations, for more information please see [the 2+ base station documentation](/docs/functional-areas/lighthouse/multi_base_stations.md).
+The lighthouse V2 protocol supports more than 2 base stations, up to 4 in the standard firmware. There is currently experimental support implemented for 4+ base stations, for more information please see [the 4+ base station documentation](/docs/functional-areas/lighthouse/multi_base_stations.md).
```

Comment:
Ah yes, but we haven't tested it in real life successfully right? Just to prevent from making too solid promises and the fact we have to configure this in the firmware by kbuild. Perhaps there is another word than experimental that we can indicate this?

Reply:
I'm pretty sure the software works with more than 4. I did test in multiple rooms, in the kitchen. There were some problems in estimating the geometry, but I think that was mainly related to poor geometry in the system. 
Maybe leave the experimental for now.

Reply:
Or we can do the 'more or less' like in your previous suggestion